test/bookstore/gke/nginx.conf

```diff
@@ -42,7 +42,7 @@ http {
       # Begin Endpoints v2 Support
       endpoints {
         on;
-        server_config /etc/nginx/endpoints/server_config.pb.txt;
+        server_config /etc/nginx/server_config.pb.txt;
```

Comment:
Just revert this change:  https://github.com/cloudendpoints/esp/commit/2f75b9819b9d78e8235bdef2af9ab7ee72fc79c6